File: 프로그래머스/lv0/120829. 각도기/각도기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
// 예각<90 직각=90 둔각>90 평각=180
int solution(int angle) {
    int answer = 0;
    
    if(angle < 90){
        answer = 1;
    } else if (angle == 90){
        answer = 2;
    } else if (angle < 180){ // 91~180 미만까지
        answer = 3;
    } else if (angle == 180){
        answer = 4;
    }
    return answer;
}

File: 프로그래머스/0/120829. 각도기/각도기.java
class Solution {
    public int solution(int angle) {
        // 예각<90 직각=90 둔각<180 평각=180
        return (angle < 90) ? 1 
             : (angle == 90) ? 2 
             : (angle < 180) ? 3 
             : 4;
    }
}